dtests/src/test/java/io/snappydata/hydra/local.smartConnectorMode.conf

```diff
@@ -0,0 +1,12 @@
+hydra.HostPrms-hostNames =
+fcn "hydra.TestConfigFcns.pool(\"<host1> <host2> <host3>\", ${snappyStoreHosts})" ncf
+fcn "hydra.TestConfigFcns.pool(\"<host1> <host2> \", ${leadHosts})" ncf
+fcn "hydra.TestConfigFcns.pool(\"<host1> <host2> <host3>\", ${locatorHosts})" ncf
+fcn "hydra.TestConfigFcns.pool(\"<host1> <host2> <host3>\", ${workerHosts})" ncf
+;
+
+io.snappydata.hydra.cluster.SnappyPrms-serverLauncherProps = " -heap-size=4g -memory-size=8g -critical-heap-percentage=95 -J-XX:+UnlockDiagnosticVMOptions  -J-XX:ParGCCardsPerStrideChunk=32768 -member-timeout=60000 -J-Dp2p.socket_timeout=60000 ";
+io.snappydata.hydra.cluster.SnappyPrms-locatorLauncherProps = " -conserve-sockets=false -J-Xloggc:gc.log -J-XX:+PrintGCDetails -J-XX:+PrintGCDateStamps -J-XX:+PrintGCTimeStamps -J-XX:+PrintHeapAtGC -member-timeout=60000 -J-Dp2p.socket_timeout=60000 ";
```

Comment:
Remove gc flags from the configuration and instead provide below parameter for this.

io.snappydata.hydra.cluster.SnappyPrms-enableGCFlags = true;

Also, remove -conserve-sockets=false parameter from the list as by default its set to false if not provided.